modules/pyclone_vi/1.0/config/default.yaml

```diff
@@ -0,0 +1,64 @@
+lcr-modules:
+
+    pyclone_vi:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            sample_maf: "__UPDATE__"
+            sample_subclones: "__UPDATE__"
```

Comment:
Can the commented example of file path with wild cards be used here, like in the phylowgs config?

Reply:
Done.